src/components/shared/SearchCard/SearchCardAction/RequestExperimentForm.js

```diff
@@ -21,17 +28,24 @@ export const RequestExperimentForm = ({ accessionCode, closeForm }) => {
           pediatric_cancer: '',
           approach: '',
           email: '',
-          email_updates: false
+          email_updates: false,
+          request_type: 'experiment'
         }}
         validationSchema={RequestDataFormSchema}
         validateOnChange={false}
         onSubmit={async (values, { setSubmitting }) => {
-          // TEMP
-          await new Promise((resolve) => {
-            setTimeout(() => resolve(values), 1000)
+          const response = await requestData({
+            requestValues: {
+              ...values
+            }
           })
-          // eslint-disable-next-line no-console
-          console.log(values)
+
+          // adds the requested experiment's accession code if not 500
+          if (response.status !== 500) {
+            addRequestedExperiment()
+          }
+
+          closeForm()
```

Comment:
If we pass in the `closeForm` as a property, we should have the prop default to noop. I think it makes more sense to call it `onSubmit`

Reply:
See the commit at 9913608ddfd2aa973a3fb2439ef7c9281d3c2689.